src/Symfony/Component/Validator/Violation/ConstraintViolationBuilder.php

```diff
@@ -33,13 +33,13 @@ class ConstraintViolationBuilder implements ConstraintViolationBuilderInterface
     private mixed $invalidValue;
     private string $propertyPath;
     private TranslatorInterface $translator;
-    private ?string $translationDomain;
+    private string|false|null $translationDomain;
```

Comment:
shouldn't we allow `false` in the argument type in the setter as well ?

Reply:
or maybe have a `->disableTranslation()` method setting it to `false`

Reply:
adding `disableTranslation()` can be done in a BC friendly way indeed